admin/schroot/Makefile

```diff
@@ -0,0 +1,113 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=schroot
+PKG_REV:=2ba1e043dbd65c8f08d1d4821393b099af05af15
+PKG_VERSION:=1.6.10
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://salsa.debian.org/debian/schroot
+PKG_MIRROR_HASH:=4df3adb05e24611f124376f2dd708413751535caf1d1d1b1acbd91d712ea843d
+PKG_SOURCE_PROTO:=git
```

Comment:
Why can't we use the packaged tarball?
http://deb.debian.org/debian/pool/main/s/schroot/schroot_1.6.10.orig.tar.xz